lib/vtls/openssl.c

```diff
@@ -2602,7 +2602,14 @@ static CURLcode ossl_connect_step1(struct Curl_easy *data,
     if(ssl_authtype == CURL_TLSAUTH_SRP)
       return CURLE_SSL_CONNECT_ERROR;
 #endif
+    /*
+     * Some OpenSSLs consider SSLv3 as deprecated but we know it better in this
+     * case!
+     */
+#pragma GCC diagnostic push
```

Comment:
you don't need a comment, the code implies what it does. i don't know whether other compilers are going to choke on the #pragmas gcc. which is actually pretty particular since it's deprecated regardless of compiler. that said i think it's unnecessary.